lookout/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for lookout.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount:
+  lookout: 1
+  dummyAnalyzer: 1
+
+image:
+  lookout:
+    repository: srcd/lookout
+    #tag: latest
+    pullPolicy: IfNotPresent
+  dummyAnalyzer:
+    repository: smacker/dummy-analyzer
```

Comment:
A noted in https://github.com/src-d/issues-infrastructure/issues/213#issuecomment-413472338, let's change this `srcd/lookout-dummy-analyzer`

Reply:
fixed in c495d7a